src/sync-controller.js

```diff
@@ -47,15 +46,7 @@ export const syncPointStrategies = [
         if (segment.dateTimeObject) {
           let segmentTime = segment.dateTimeObject.getTime() / 1000;
           let segmentStart = segmentTime + syncController.datetimeToDisplayTime;
-          let distance = Math.abs(currentTime - segmentStart);
-
-          // Once the distance begins to increase, we have passed
-          // currentTime and can stop looking for better candidates
-          if (lastDistance !== null && lastDistance < distance) {
-            break;
```

Comment:
I am not sure if this is the correct fix, but I don't have a video to test with for the code that this was implemented for. Basically without this break I the syncPoint appears correct and with it the syncPoint is always `{time: 0, segmentIndex: 0}`